### PR TITLE
feat: new onboarding step

### PR DESCRIPTION
**Pull request by danil-pavlov**

### CODE REVIEW

File: docs/topics/multiplatform-mobile/multiplatform-mobile-update-ui.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+       <img src="icon-5-todo.svg" width="20" alt="Fifth step"/> Share more logic<br/>
+       <img src="icon-6-todo.svg" width="20" alt="Sixth step"/> Wrap up your project</p>
+</microformat>
+
+To build the user interface, you'll use the [Jetpack Compose](https://developer.android.com/jetpack/compose) toolkit
+for the Android part of your project and [SwiftUI](https://developer.apple.com/xcode/swiftui/) for the iOS one.
+These are both declarative UI frameworks, and you'll see similarities in the UI implementations. In both cases,
+you store the data in the `phrases` variable and later iterate over it to produce a list of `Text` items.
+
+### The Android module
```
Comment (koshachy) [inline docs/topics/multiplatform-mobile/multiplatform-mobile-update-ui.md:18]: I expect here a verb. Like "update the android module" or something similar

File: docs/kr.tree [hunk 2]
```diff
@@ -129,9 +129,10 @@
             <toc-element id="multiplatform-mobile-getting-started.md" accepts-web-file-names="kmm-getting-started.html"/>
             <toc-element id="multiplatform-mobile-setup.md" accepts-web-file-names="kmm-setup.html" toc-title="1. Set up an environment"/>
             <toc-element id="multiplatform-mobile-create-first-app.md" accepts-web-file-names="kmm-create-first-app.html" toc-title="2. Create your first cross-platform app"/>
-            <toc-element id="multiplatform-mobile-dependencies.md" toc-title="3. Add dependencies"/>
-            <toc-element id="multiplatform-mobile-upgrade-app.md" toc-title="4. Upgrade your app"/>
-            <toc-element id="multiplatform-mobile-wrap-up.md" toc-title="5. Wrap up your project"/>
+            <toc-element id="multiplatform-mobile-update-ui.md" toc-title="3. Update UI"/>
```
Comment (koshachy) [inline docs/kr.tree:132]: I'm thinking about expanding the toc-title to make it more readable.
In the nav tree the titles should be short (like "Share more logic"), but in the title of the content I'd like to see something like "Share more logic between iOS and Android").

What do you think about this?

**Review by koshachy — APPROVED** [review]
Some minor comments. Please resolve them before proceeding to the publishing.

In some sections sentences are a bit complicated, I'd like to divide them into two-three simple sentences.But overall it's a good tutorial, I really like it. 

Great job!

File: docs/topics/multiplatform-mobile/multiplatform-mobile-create-first-app.md [hunk 3]
```diff
@@ -61,72 +60,237 @@ Each Kotlin Multiplatform project includes three modules:
...
+    interface Platform {
+        val name: String
+    }
+    ```
+   
+   It's a common `Platform` interface with information about the platform.
+2. Switch between the `androidMain` and the `iosMain` modules.
+   You'll see that they have different implementations of the same functionality for the Android and the iOS source sets:
+    
+    ```kotlin
```
[the remark this answers is not in the document]
Comment (danil-pavlov) [reply docs/topics/multiplatform-mobile/multiplatform-mobile-create-first-app.md:137]: Totally! we already started to do this in some later drafts as well

File: docs/kr.tree (hunk 2 above)
Comment (danil-pavlov) [reply docs/kr.tree:132]: Makes sense, I got rid off the acronum and expanded the header for the 5th step

**Pull request merged**